### PR TITLE
[FAI-7813] Chart Explainer UI for user scenario 1

### PR DESCRIPTION
**Pull request by kinfaikan**
- create popover for displaying chart explanation
- call lighthouse ai for chart explanation via alastor
- put Chart Explainer behind a feature flag

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -24,21 +24,36 @@ RUN INTERACTIVE=false CI=true MB_EDITION=$MB_EDITION bin/build.sh
 ## jar from the previous stage rather than the local build
 ## we're not yet there to provide an ARM runner till https://github.com/adoptium/adoptium/issues/96 is ready
 
-FROM --platform=linux/amd64 eclipse-temurin:11-jre-alpine as runner
+#FROM --platform=linux/amd64 eclipse-temurin:11-jre-alpine as runner
+FROM eclipse-temurin:11-jre-jammy as runner
```
Comment (chalenge) [inline Dockerfile:28]: do we really need these changes or they were need for your local dev?

File: bin/build/src/build.clj [hunk 2]
```diff
@@ -36,13 +36,7 @@
                :env {"PATH"       (env/env :path)
                      "HOME"       (env/env :user-home)
                      "WEBPACK_BUNDLE"   "production"
-                     "MB_EDITION" mb-edition
```
Comment (chalenge) [inline bin/build/src/build.clj:39]: think you need to revert these changes

**Review by chalenge — COMMENTED** [review]
@kinfaikan Looks like you need to revert some changes maybe need for local dev.

File: package.json [hunk 3]
```diff
@@ -13,8 +13,6 @@
     "@emotion/react": "^11.7.1",
     "@emotion/styled": "^11.6.0",
     "@react-oauth/google": "^0.4.0",
-    "@sentry/react": "^7.29.0",
```
[the remark this answers is not in the document]
Comment (kinfaikan) [reply package.json:16]: The Metabase UI does not come up (at least locally) because of the sentry stuff. We talked about that some time ago. Any suggestion?

https://faros-ai.slack.com/archives/GPB79LS81/p1693003643494019?thread_ts=1692835531.134169&cid=GPB79LS81

File: Dockerfile (hunk 1 above)
Comment (kinfaikan) [reply Dockerfile:28]: My bad. This is for local dev on mac M2.

Comment (kinfaikan) [reply Dockerfile]: reverted

**Review by chalenge — COMMENTED** [review]
LGTM

**Review by chalenge — APPROVED** [review]

File: frontend/src/metabase/components/Popover/Popover.css [hunk 4]
```diff
@@ -215,7 +215,7 @@
 /* create a slightly larger arrow on the bottom for border purposes */
 .tether-element-attached-bottom .PopoverBody--withArrow:before {
   bottom: -20px;
-  border-top-color: var(--color-border);
+  border-top-color: var(--background-dark, #023d67);
```
[the remark this answers is not in the document]
Comment (kinfaikan) [reply frontend/src/metabase/components/Popover/Popover.css:218]: Good catch! Afaik, MB doesn't use popover with arrow at the moment. Maybe we can fix this in a follow-up PR?

File: frontend/src/metabase/visualizations/components/ScalarValue/ScalarValue.jsx [hunk 5]
```diff
@@ -57,30 +59,175 @@ const ScalarValue = ({
...
+  const explanationPopover = (
+    <div
+      style={{
+        display: "flex",
+        padding: "16px",
+        flexDirection: "column",
+        alignItems: "flex-start",
+        alignSelf: "stretch",
+        borderRadius: "4px",
+        background: "var(--background-dark, #023D67)",
```
[the remark this answers is not in the document]
Comment (kinfaikan) [reply frontend/src/metabase/visualizations/components/ScalarValue/ScalarValue.jsx:125]: yes, the color code is from Tiffany's design.

**Review by chalenge — APPROVED** [review]
